### DISCUSSION TITLE
Renew Canva after 12 months

### DISCUSSION THREAD

**Discussion by Annatimteo** [community/community#39273] [GitHub Education]
I would like to know what steps to i have to take to renew Canva for Education after 12 months of use.

**Comment by kodepirate** [comment]
Hey @Annatimteo 
I believe that your Canva will not update after expiring, The student pack renews means you can re-use your student pack again after 12 months but your the student pack will not be renewed..

**Reply by Annatimteo** [reply]
I am kinda confused, you said that the student pack it's renewable but then said that it will not be renewed, you meant that the Canva will not be renewed along with the pack?

**Comment by kodepirate** [comment]
Yes talking about that one, those stuffs which you used will not be renewed @Annatimteo

**Comment by dhrubasaha08** [comment]
@Annatimteo Unfortunately Canva for Education is no longer offered by GitHub in their Student Developer Pack !

**Comment by kodepirate** [comment]
Yeah sadly

**Comment by AAR2020advik** [comment]
It is not included in the Student Dev Pack. You can basically use the student pack that was old (and has canva) (IDK  if it would work) but you cannot re renew it so it will not be usable after 1 month

**Comment by samus-aran** [comment]
Hi @Annatimteo  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for their 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
